telematic_system/docker-compose.historical.yml

```diff
@@ -27,6 +27,12 @@ services:
     - INFLUX_ORG=my-org
     - INFLUX_TOKEN=my-super-secret-token
     - INFLUX_URL=<>
+    - MYSQL_HOST=telematic_system_mysqldb_1 # Add host IP where mysql database is hosted
```

Comment:
Replacing "telematic_system_mysqldb_1"  with <MYSQL_HOST>

Reply:
Please update the rest of the MYSQL environment variables with placeholder

Reply:
Add the historical processing service to the docker-compose.local.yml file with the preset MYSQL credentials.